indexer_app/management/commands/populatedata.py

```
@@ -0,0 +1,547 @@
+            print("adding admins")
+            for admin in config["admins"]:
+                admin_obj, _ = Account.objects.get_or_create(defaults={"chain_id":1},id=admin)
+                pot.admins.add(admin_obj)
+            # add activity
+            activity_defaults = {
+                "signer_id": owner.id,
+                "receiver_id": POTFACTORY_ID,
+                "timestamp": datetime.fromtimestamp(deployed_at_ms / 1000),
+                "tx_hash": None,
+            }
+            activity, _ = Activity.objects.update_or_create(
```

Ruff (0.14.1)

Unpacked variable `activity` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)